app/serializers/v1/invoice_custom_section_serializer.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module V1
+  class InvoiceCustomSectionSerializer < ModelSerializer
+    def serialize
+      {
+        lago_id: model.id,
+        code: model.code,
+        name: model.name,
+        description: model.description,
+        details: model.details,
+        display_name: model.display_name,
+        selected_for_organization: model.selected_for_organization?,
+        organization: {
+          lago_id: model.organization_id
+        }
```

Comment:
Do we plan to add more organization details in this serializer? If no I would suggest to flatten it (not a blocker ;) )

Reply:
yep, makes perfect sense, updating, thank you!

Reply:
(updated it in the last PR of the chain: https://github.com/getlago/lago-api/pull/3013/files)